cluster/examples/workloads/kubernetes/wordpress/azure/provider.yaml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
I believe we've in some cases been deduplicating the various provider examples in favor of the YAMLs under `/cluster/examples/providers`. Should we do that here?

Reply:
I am open to either way. In adding this example, I tried to mirror the AWS and GCP existing examples as much as possible.

Reply:
I'd prefer to avoid duplicating these YAMLs where possible; it only increases the number of examples we have to manually find and update if/when the shape of their API types change. I suggest we update Azure use the shared providers.